src/test/java/org/asciidoctor/ant/extensions/TwitterMacro.java

```diff
@@ -15,28 +15,32 @@
  */
 package org.asciidoctor.ant.extensions;
 
-import org.asciidoctor.ast.AbstractBlock;
-import org.asciidoctor.ast.Inline;
+import org.asciidoctor.ast.ContentNode;
+import org.asciidoctor.ast.PhraseNode;
 import org.asciidoctor.extension.InlineMacroProcessor;
 
 import java.util.HashMap;
 import java.util.Map;
 
 public class TwitterMacro extends InlineMacroProcessor {
 
+    public TwitterMacro(String macroName) {
+        super(macroName);
+    }
+
     public TwitterMacro(String macroName, Map<String, Object> config) {
         super(macroName, config);
     }
 
     @Override
-    protected Object process(AbstractBlock parent, String twitterHandle, Map<String, Object> attributes) {
+    public Object process(ContentNode parent, String twitterHandle, Map<String, Object> attributes) {
         // Define options for an 'anchor' element.
         Map<String, Object> options = new HashMap<String, Object>();
         options.put("type",":link");
         options.put("target", "http://www.twitter.com/@"+twitterHandle);
 
         // Prepend twitterHandle with @ as text link.
-        final Inline inlineTwitterLink = createInline(parent, "anchor", "@"+twitterHandle, attributes, options);
+        final PhraseNode inlineTwitterLink = createPhraseNode(parent, "anchor", "@"+twitterHandle, attributes, options);
```

Comment:
I based these changes on the ManpageMacro present in the asciidoctorj repository.

See https://github.com/asciidoctor/asciidoctorj/blob/asciidoctorj-1.6.0/asciidoctorj-core/src/test/java/org/asciidoctor/extension/ManpageMacro.java .